File: main/dto/TeamDto.cs
using System.Text.Json.Serialization;

namespace main.dto
{
    public class TeamDto
    {
        [JsonPropertyName("id")]
        public long? Id { get; set; }
        [JsonPropertyName("name")]
        public string Name { get; set; }
        [JsonPropertyName("city")]
        public City City { get; set; }
    }
}

File: java-client/src/main/java/cs/ubb/mpp/client/dto/TeamDto.java
package cs.ubb.mpp.client.dto;

import java.io.Serializable;

public class TeamDto {
    private Long id;
    private String name;
    private City city;

    public TeamDto(Long id, String name, City city) {
        this.id = id;
        this.name = name;
        this.city = city;
    }

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public City getCity() {
        return city;
    }

    @Override
    public String toString() {
        return "TeamDto{" +
                "name='" + name + '\'' +
                ", city=" + city +
                '}';
    }
}
